cmd/helms3/main.go

```diff
@@ -63,6 +63,8 @@ func main() {
 	initURI := initCmd.Arg("uri", "URI of repository, e.g. s3://awesome-bucket/charts").
 		Required().
 		String()
+	initACL := initCmd.Flag("acl", "Sets the ACL for the object when the chart is pushed, e.g. bucket-owner-full-control").Default("").OverrideDefaultFromEnvar("S3_ACL").
```

Comment:
It seems that all commands can use this flag. What do you think about moving it to a global flag definition (like `--timeout`)?

Reply:
Also I think we should mention about the index file too, not only charts.